### PR TITLE
Test verbosity

### PR DESCRIPTION
**Pull request by PiRK**
Addresses #136

My current change https://github.com/silx-kit/silx/commit/d6cbd0334e7583332828443b2a5def12b6607a02 does not modify the previous behavior of printing dots for successful tests, `s` for skipped tests, `E` for errors and `F` for failures.  It just adds the option to increase the verbosity of the tests themselves (not only the logging verbosity) with option -vv.

The test verbosity parameter can be:

- 0 (quiet): you just get the total numbers of tests executed and the global result
- 1 (default): you get the same plus a dot for every successful test or a F for every failure
- 2 (verbose): you get the help string of every test and the result

To summarize:
 
- `python run_tests.py` sets `logging` level to _WARNING_ and `unittest.TextTestRunner` verbosity to _1_
- `python run_tests.py` sets `logging` level to _INFO_ and `unittest.TextTestRunner` verbosity to _1_
-  `python run_tests.py` sets `logging` level to _DEBUG_ and `unittest.TextTestRunner` verbosity to _2_

### CODE REVIEW

**Comment by t20100** [comment]
The -vv is... verbose ;)

What about the following patch, which adds some logs with the names, docstring and reason of skipped tests?

```
-if runner.run(test_suite).wasSuccessful():
+result = runner.run(test_suite)
+for test, reason in result.skipped:
+    logger.warning('Skipped %s (%s): %s',
+                   test.id(), test.shortDescription() or '', reason)
+
+if result.wasSuccessful():
```

Displays, e.g.,
`WARNING:run_tests:Skipped silx.io.test.test_specfile.TestSFLocale.test_locale_de_DE (): de_DE.utf8 locale not installed`

**Comment by PiRK** [comment]
Good idea. I added it to the PR, in addition to `-vv`in case someone wants a complete listing of everything that happened.

**Comment by t20100** [comment]
Looks good to me
